tutor/src/screens/task/step/instructions.js

```diff
@@ -21,6 +21,16 @@ const Header = styled.h2`
   line-height: inherit;
   justify-content: space-between;
   align-items: center;
+
+  ${({ theme }) => theme.breakpoint.tablet`
+    > span > span {
+      display: none;
```

Comment:
since we control the html layout, how about adding a class to the element that can be hidden instead of writing nested rules like this.  That seems less fragile to me